en/docs/install-and-setup/setup/api-controller/managing-microgateways/managing-microgateways-with-ctl.md

```diff
@@ -0,0 +1,296 @@
+        ```bash
+        apictl mg get apis -e dev --query type:http -k
+        ```
+
+    !!! info
+        **Flags:**
+        
+        - Mandatory :    
+            `-e`, `--environment` : Microgateway adapter environment to list the APIs from  
+
+        - Optional :       
+            `-l`, `--limit` : Maximum number of APIs to return     
+            `-q`, `--query` : Query to filter the APIs
+
+    !!! example
+        ```bash
+        apictl mg get apis -e dev -k
+        ```
+        ```bash
+        apictl mg get apis -q type:http --environment dev --limit 100 -k
+        ```
+        ```bash
+        apictl mg get apis -q type:ws -e dev -l 10 -k
+        ```
+
+-   **Response**
+
+    ```
+    Limit flag not set. Set to default: 25
+    APIs total: 2 received: 2
+    NAME                VERSION             TYPE                CONTEXT         GATEWAY_ENVS
+    SwaggerPetstore     1.0.5               HTTP                /v2             [Production and Sandbox]
+    SwaggerPetstore1    1.0.5               WS                  /v2             [Production and Sandbox]
+    ```
+
+
+## Undeploy an API
+
+This lets you remove an API from the microgateway. 
+
+-   **Command**
+    ```bash
+    apictl mg undeploy api -e <environment-name> -n <api-name> -v <api-version> -k
+    ```
+    ```bash
+    apictl mg undeploy api -e <environment-name> -n <api-name> -v <api-version> --vhost <vhost> -k
+    ```
+
+    !!! info
+        **Flags:**
+        
+        - Mandatory :    
+            `-e`, `--environment` : Microgateway adapter environment to undeploy the API from   
+            `-n`, `--name`        : API name   
+            `-v`, `--version`     : API version   
+
+        - Optional :     
+            `-t`, `--vhost` : Virtual host the API belongs to   
+
+    !!! example
+        ```bash
+        apictl mg undeploy api -e dev -n petstore -v 0.0.1 -k
+        ```
+        ```bash
+        apictl mg undeploy api --name SwaggerPetstore --version 0.0.1 -e dev -k
+        ```
+        ```bash
+        apictl mg undeploy api -n petstore -v 0.0.1 -e dev --vhost www.pets.com
+        ```
+-   **Response**
+
+    ```
+    API undeployed from microgateway successfully!
+    ```
+
+## Log out from a Microgateway Environment
+
+This will erase the access token of the microgateway adapter from apictl.
+
+-   **Command**
+    ```bash
+    apictl mg logout <environment-name>
+    ```
+
+    !!! example
+        ```bash
+        apictl mg logout dev
+        ```
+-   **Response**
+
+    ``` bash tab="Response Format"
+    Logged out from Microgateway Adapter in environment: <environment-name>
+    ```
+
+    ``` bash tab="Example Response"
+    Logged out from Microgateway Adapter in environment: dev
+    ```
+
+## Remove a Microgateway Environment
+
+This will delete the adapter url and its access token first, and then remove the microgateway environment from apictl.
```

Comment:
Shall we change the intro like this ?
```suggestion
This command uses to remove a microgateway environment from the apictl. Once this command is called, it will delete the adapter url and its access token first, and then remove the microgateway environment from apictl.
```

Reply:
"This command can be used to remove a microgateway environment from the apictl. Once this command is executed, it will delete the adapter url and its access token first, and then remove the microgateway environment from apictl."
WDYT?

Reply:
+1

Reply:
Updating the intro for remove env